101-VM-Virus-Attack-Prevention/azuredeploy.parameters.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "location": {
+        "value": "East US"
+      },
+      "omsSku": {
```

Comment:
remove parameters that have defaultValues in the template

Reply:
removed parameters having default values in template.